config/boards/Edge.conf

```diff
@@ -31,7 +31,7 @@ case "$LINUX" in
 		LINUX_DTB="arch/arm64/boot/dts/rockchip/rk3399-khadas-edge-linux.dtb"
 		LINUX_DIR="$ROOT/linux"
 		LINUX_GIT_BRANCH="khadas-edge-4.4.y"
-		LINUX_DEFCONFIG="kedge_defconfig"
+		LINUX_DEFCONFIG="${LINUX_DEFCONFIG:- kedge_defconfig}"
```

Comment:
You want to use other configurations for linux building?

Reply:
yes ! for some reasons we must have something like this ;)

Reply:
???